modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -608,10 +607,10 @@ open class PlaybackManager @Inject constructor(
                 onCompletion(episode.uuid)
             }
         }
-        trackPlayback(AnalyticsEvent.PLAYBACK_SKIP_FORWARD)
+        trackPlayback(AnalyticsEvent.PLAYBACK_SKIP_FORWARD, playbackSource)
     }
 
-    fun skipBackward() {
+    fun skipBackward(playbackSource: PlaybackSource = PlaybackSource.UNKNOWN) {
```

Comment:
Just food for thought... I have mixed feelings about defaulting `PlaybackSource` arguments to be `UNKNOWN`. I kind of like the idea of not having a default so that it feels kind of icky everytime you see a call with an `UNKNOWN` (or <gasp> add a call with an `UNKNOWN`) and helps give us a nudge to get rid of those unknowns when we notice that one doesn't make sense. Having it just be a default paramger kind of feels like it becomes hidden.

On the other hand, losing the default parameter adds quite a bit of boilerplate. Like I said, I feel conflicted, and I just wanted to share my thinking. This is totally fine as-is.